src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-create-initial/index.js

```diff
@@ -4,27 +4,63 @@ import template from './sw-order-create-initial.html.twig';
  * @package checkout
  */
 
-const { State } = Shopware;
+const { State, Data, Service } = Shopware;
+const { Criteria } = Data;
 
 // eslint-disable-next-line sw-deprecation-rules/private-feature-declarations
 export default {
     template,
 
     compatConfig: Shopware.compatConfig,
 
+    inject: ['feature'],
+
+    computed: {
+        customerRepository() {
+            return Service('repositoryFactory').create('customer');
+        },
+
+        defaultCriteria() {
+            const criteria = new Criteria(1, 25);
+            criteria
+                .addAssociation('addresses')
+                .addAssociation('group')
+                .addAssociation('salutation')
+                .addAssociation('salesChannel')
+                .addAssociation('lastPaymentMethod')
+                .addAssociation('defaultBillingAddress.country')
+                .addAssociation('defaultBillingAddress.countryState')
+                .addAssociation('defaultBillingAddress.salutation')
+                .addAssociation('defaultShippingAddress.country')
+                .addAssociation('defaultShippingAddress.countryState')
+                .addAssociation('defaultShippingAddress.salutation')
+                .addAssociation('tags');
+
+            if (!this.feature.isActive('v6.7.0.0')) {
+                criteria.addAssociation('defaultPaymentMethod');
+            }
+
+            return criteria;
+        },
+    },
+
     created() {
         this.createdComponent();
     },
 
     methods: {
-        createdComponent() {
-            const { customer } = this.$route.params;
+        async createdComponent() {
+            const { customerId } = this.$route.query;
 
-            if (!customer) {
+            if (!customerId) {
                 return;
             }
 
-            State.commit('swOrder/setCustomer', customer);
+            try {
+                const customer = await this.customerRepository.get(customerId, Shopware.Context.api, this.defaultCriteria);
+                State.commit('swOrder/setCustomer', customer);
+                // eslint-disable-next-line no-empty
+            } catch (error) {}
```

Comment:
Don't catch any errors silently, especially if no error is excepted. Also, only set the customer if one is found:
```suggestion
            const customer = await this.customerRepository.get(customerId, Shopware.Context.api, this.defaultCriteria);
            if (customer) {
                State.commit('swOrder/setCustomer', customer);
            }
```

Reply:
Took it over, thanks!